PowerShell/ScubaGear/Testing/Unit/PowerShell/Providers/EXOProvider/Get-ScubaDkimRecord.Tests.ps1

```diff
@@ -0,0 +1,101 @@
+$ProviderPath = "../../../../../Modules/Providers"
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "$($ProviderPath)/ExportEXOProvider.psm1") `
```

Comment:
Although outside the scope of this issue, I wonder if there is a way we can abstract this logic into a helper function? Since we are hard coding paths like this, if we ever change the path, we would need to update a bunch of files that have this code in it.

Reply:
Agree about that being out of scope, but that would certainly be a valuable enhancement to the unit tests.